### DISCUSSION TITLE
新版本播放页面？

### DISCUSSION THREAD

**Discussion by FoundTheWOUT** [the1812/Bilibili-Evolved#2761] [综合讨论 / General]
![FireShot Capture 001 - 丛云光弹踢罐也太爽了吧_哔哩哔哩_bilibili - www bilibili com](https://user-images.githubusercontent.com/47922588/147080096-d9847df9-c0b9-45eb-828b-9522527b2999.png)

**Comment by Geeyun-JY3** [comment]
和现在稳定版相比改动相对较小的一版？

感觉改得好频繁，我看过好几版不同的图了（虽然一次都没内测到我

**Comment by the1812** [comment]
我居然收到新版视频页了, 虽然还是想先做简化首页(
